code/modules/unit_tests/knockoff_component.dm

```diff
@@ -0,0 +1,86 @@
+/// Helper to reset the glasses dummy (wears_the_glasses) back to it's original position, clear knockdown, and return glasses (if gone)
+#define SET_GLASSES_WEARER \
```

Comment:
`#undef` at the end, but in my opinion this should just be a proc, and the context variables it uses should either be parameters or variables on src. I don't really like unhygienic macros like these